### ISSUE TITLE
Training Error

### ISSUE THREAD

**Issue by sammyfreg** [buliasz/tesstrain-windows-gui#2]
Hi, thank you for this tool. I'm new with Tesseract and just wanted to try recognizing some digits, which naive me thought that it wouldn't be too much trouble, I was wrong :)

First item I should mention is that 'Ground Truth Preview' doesn't like the default install path of Tesseract on Windows. There's a space in "Program Files\Tesseract" and Tesstrain-win-gui isn't using " " when launching the associated exe. So I installed it in a path without spaces in the name. 

So, my user case is that I have a bunch of images, with some GPS coordinates printed on them. I pre-processed them with ImageMagicks to only contain the relevant information, an try extracting the digits. Unfortunately, my results might only be 75% accurate, leading me to trying to train the system for this.

I am able to train some data without any start models using your UI. My error rate is higher than using the 'Eng' model, so I thought I would try starting from the 'eng' model. Unfortunatly, it generates some error that I would like some help with. Would it be possible to point me in the right direction ?

Thank you.

Here is the error I get:
<img width="1237" alt="OutputError" src="https://user-images.githubusercontent.com/19142627/158590607-ab34ac15-d682-41b2-9b7d-c6a27746166f.png">

I included my training data set (50 images) and the output of each step until the error, when 'debug' option is selected.
[manholetrainset.zip](https://github.com/buliasz/tesstrain-windows-gui/files/8262092/manholetrainset.zip)
[OutputSteps.zip](https://github.com/buliasz/tesstrain-windows-gui/files/8262112/OutputSteps.zip)

**Comment by WhosVenom** [comment]
Having the same issue, scoured the internet and came out empty-handed.

Very frustrating.

**Comment by sammyfreg** [comment]
I seem to also have the same issue with the ocrd-testset, so not an issue with my data after all. :/ Maybe the default installed eng.traineddata is not good for this? At some point, I tried with other traineddata, but was still getting an error.

**Comment by buliasz** [comment]
Hi, thanks @sammyfreg  for great issue report. I've just fixed bugs related to the "Program Files" installation.
Now for the other part, the real cause is in the file "OutputStep6Error.png" you have attached.
![image](https://user-images.githubusercontent.com/22753806/158736434-155b1942-428d-4770-836e-bf32dffd5068.png)
What it means is you have used "fast" model which doesn't have data required to be used as starting model. 
If you go Tesseract User Manual, specifically [this part](https://tesseract-ocr.github.io/tessdoc/#traineddata-files), you can see there are two versions: "best" and "fast". You just need to download the "best" version and everything should be working good.

I'll try to detect this situation in future and make it clear in GUI that this is wrong version of trained data, when I get some free time for this.

**Comment by sammyfreg** [comment]
Thank you, that was it!

I thought I had tried the 'best' model at some point, with still some errors of some kind, but now the training seems to be happening properly.

For other people reference, you can find the 'best' trained set [here](https://github.com/tesseract-ocr/tessdata_best) :

**Comment by sammyfreg** [comment]
Just a quick update. 

For a long time, I thought my training results were still poor, since when using `Preview`, it would report a lot of errors, with the `Find Next Wrong` button. Then, I guessed that maybe it was using the 'Start Model' for the test, and not the newly trained model. changing the 'Start Model' the newly trained/copied one, I suddenly can confirm that my results are a lot better! So, maybe this should try to pick the last trained model, or show another dropdown to select the trained data used for the Preview? 

I managed to get some decent results after training, thanks for this tool that made it easier than trying to figure out all the details.
